utils/helpers/constants/key-list.js

```diff
@@ -0,0 +1,108 @@
+export const twoLevelsNestedKeyListParams = {
+  type: "keyList",
+  keys: [
+    {
+      type: "keyList",
+      keys: [
+        {
+          type: "ecdsaSecp256k1PublicKey",
+        },
+        {
+          type: "ecdsaSecp256k1PrivateKey",
+        },
+      ],
+    },
+    {
+      type: "keyList",
+      keys: [
+        {
+          type: "ecdsaSecp256k1PublicKey",
+        },
+        {
+          type: "ed25519PublicKey",
+        },
+      ],
+    },
+    {
+      type: "keyList",
+      keys: [
+        {
+          type: "ed25519PrivateKey",
+        },
+        {
+          type: "ecdsaSecp256k1PublicKey",
+        },
+      ],
+    },
+  ],
+};
+
+export const threeKeysKeyListParams = {
+  type: "keyList",
+  keys: [
+    {
+      type: "ed25519PrivateKey",
+    },
+    {
+      type: "ecdsaSecp256k1PublicKey",
+    },
+    {
+      type: "ed25519PublicKey",
+    },
+  ],
+};
+
+export const fourKeysKeyListParams = {
```

Comment:
I think we can look to replace instances where `threeKeysKeyListParams` and `fivePrivateKeysKeyListParams` are used with this, since this covers all ed25519 and ecdsaSecp256k1 public and private keys

Reply:
Yes, I think so too. The `fourKeysKeyListParams` includes all key types in use. I’ve updated it and tested it, and it seems to work perfectly!